frontend/app/routes/_public+/intake+/$id+/review-information.tsx

```diff
@@ -0,0 +1,217 @@
+import type { ReactNode } from 'react';
+
+import { json, redirect } from '@remix-run/node';
+import type { ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node';
+import { Form, useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { Address } from '~/components/address';
+import { Button } from '~/components/buttons';
+import { InlineLink } from '~/components/inline-link';
+import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('gcweb');
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  //TODO: Get User/intake form information
+  const userInfo = {
+    firstName: 'John',
+    id: '00000000-0000-0000-0000-000000000000',
+    lastName: 'Maverick',
+    phoneNumber: '(754) 628-4776',
+    altPhoneNumber: '(819) 777-1234',
+    preferredLanguage: 'fr',
+    birthday: new Date().toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
+    sin: '123456789',
+    martialStatus: 'Married',
+    email: 'myemail@example.com',
+    communicationPreference: 'digital',
+  };
+  const spouseInfo = {
+    firstName: 'Phil',
+    id: '00000000-0000-0000-0000-000000000001',
+    lastName: 'Doe',
+    phoneNumber: '(754) 628-4776',
+    birthday: new Date().toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
+    sin: '123456789',
+    consent: true,
+  };
+  const preferredLanguage = {
+    id: 'fr',
+    nameEn: 'French',
+    nameFr: 'Français',
+  };
+  const mailingAddressInfo = {
+    address: '44367 Ibrahim Field',
+    city: 'Lake Granvillestead',
+    province: 'PE',
+    postalCode: 'P1L 1C5',
+    country: 'CAN',
+  };
+  const homeAddressInfo = {
+    address: '724 Mason Mission',
+    city: 'Reichelmouth',
+    province: 'NS',
+    postalCode: 'T9K 6P4',
+    country: 'CAN',
+  };
+  const dentalInsurance = {
+    private: [],
+    public: ['Dental and Opcial Assistance for Senioirs (65+)'],
+  };
+
+  return json({ userInfo, spouseInfo, preferredLanguage, homeAddressInfo, mailingAddressInfo, dentalInsurance });
+}
+
+export async function action({ request }: ActionFunctionArgs) {
+  //TODO: Add intake forn logic
+  return redirect(`/intake`);
+}
+
+export default function ReviewInformation() {
+  const { userInfo, spouseInfo, preferredLanguage, homeAddressInfo, mailingAddressInfo, dentalInsurance } = useLoaderData<typeof loader>();
+  const { i18n } = useTranslation(i18nNamespaces);
+  return (
+    <>
+      <h2 className="text-2xl font-semibold">Applicant Information</h2>
+      <dl>
+        <DescriptionListItem term="Date of birth">{userInfo.birthday}</DescriptionListItem>
+        <InlineLink id="change-date-of-birth" to="/">
+          Change date of birth
+        </InlineLink>
```

Comment:
It might be worth running this page against an [HTML Validator](https://validator.w3.org/), which would indicate that "Element [a](https://html.spec.whatwg.org/multipage/#the-a-element) not allowed as child of element [dl](https://html.spec.whatwg.org/multipage/#the-dl-element) in this context.", (among other validation issues).